__tests__/server.test.js

```diff
@@ -35,4 +35,41 @@ describe("GET /api", ()=>{
             expect(endpoints).toEqual(endpoints)
         })
     })
+})
+
+describe("GET /api/articles/:article_id", ()=>{
+    test("200: Received a Response", ()=>{
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({body: {article}})=>{
+            console.log(article)
+            expect(article).toMatchObject({
+                article_id: expect.any(Number),

```

Comment:
You're not expecting any number, you're specifically expecting id 1. Same goes for all of these keys, you are testing that the article is the one you requested in full. This needs to be changed.